styles.css

```diff
@@ -676,6 +681,10 @@
     border: 1px solid #ff6b00;
   }
 
+  .color:hover {
+    cursor: pointer;
+  }
+
   .title-line {
     width: 794px;
     height: 0;
```

Comment:
- [ ] As suggested by the previous reviewer, I would kindly suggest you work on the contract to the button's interactions:
Currently your `see project` button is failing the button contrast checker test.

The result 👇 

![but](https://user-images.githubusercontent.com/78634841/215872203-9c936bfb-aa7b-4353-a133-9e4d3a1fbbab.png)

Reply:
thank you for reviewing my PR